packages/editor-ui/src/api/versions.ts

```diff
@@ -1,6 +1,7 @@
 import { IVersion } from '@/Interface';
+import { IDataObject } from 'n8n-workflow';
 import { get } from './helpers';
 
-export async function getNextVersions(endpoint: string, version: string, instanceId?: string): Promise<IVersion[]> {
-	return await get(endpoint, version, {}, instanceId);
+export async function getNextVersions(endpoint: string, version: string, headers?: IDataObject): Promise<IVersion[]> {
```

Comment:
I would keep this as it was before... `get` was a generic function to call the api. `getNextVersions` is not generic and requires specific params to work, i.e instance id... 
```suggestion
export async function getNextVersions(endpoint: string, version: string, instanceId?: string): Promise<IVersion[]> {
```